### ISSUE TITLE
OSX Client Theremin does not work with Mopidy 0.19.4 

### ISSUE THREAD

**Issue by rawdlite** [mopidy/mopidy#867]
Theremin can not connect with Mopidy 0.19.4 .

After looking into the log files i suspect it has to do with `listallinfo` not being supported.

What did i do:

- start mopidy with extensions disabled (exept local and mpd)
- start Theremin

Theremin reports: Lost Connection with Server.

Mopidy shows:

```
INFO     New MPD connection from [::ffff:192.168.0.101]:55469
WARNING  Rejected connection from [::ffff:192.168.0.101]:55480
```

Last entry in log file was:

```
DEBUG    2014-10-06 14:41:36,430 [15957:MpdSession-12] mopidy.mpd.session
  Request from [::ffff:192.168.0.101]:55113: stats
DEBUG    2014-10-06 14:41:36,431 [15957:MpdSession-12] mopidy.mpd.session
  Response to [::ffff:192.168.0.101]:55113:
    uptime: 0
    db_update: 0
    artists: 0
    playtime: 0
    albums: 0
    db_playtime: 0
    songs: 0
    OK
DEBUG    2014-10-06 14:41:36,442 [15957:MpdSession-12] mopidy.mpd.session
  Request from [::ffff:192.168.0.101]:55113: listallinfo "/"
```

```
$ mpc -p 6602 listallinfo "/"
unknown command "listallinfo"
Usage: mpc [options] <command> [<arguments>]
mpc version: 0.25
```

**Comment by adamcik** [comment]
#990 likely "fixed" this - by fixed I mean that we've disabled the `listallinfo` command, so Theremin shouldn't hang on waiting for the repsonse. But that also means we might have broken the client by taking away support for that command.

Not sure if there is anything more we can really do for this issue?

**Comment by jodal** [comment]
I tried to reproduce this issue using the updated fork of Theremin from https://github.com/TheStalwart/Theremin.

- Mopidy 0.19.5 with Dirble: `lsinfo "/"` returns a quite quickly response (due to timeout error in Dirble), but Theremin disconnects.
- Mopidy 0.19.5 without any extensions: Theremin connects and stays connected when playing a stream added with `mpc add ...`.

Note that this Theremin version seems to use `lsinfo "/"`, which only lists the files/dirs at the root level, and not `listallinfo "/`"` which lists all files/dirs recursively. This sounds like a major improvement.

In conclusion:

- Mopidy 0.20 should make the original Theremin client work better (or not at all) by disabling support for `listallinfo`.
- The more up to date Theremin fork does no longer use `listallinfo` at connect time, working around this.
